bot.py

```diff
@@ -264,18 +382,6 @@ def riyans_denial(response):
             return True
     return False
 
-
-def new_user_message(response):
```

Comment:
All i'm saying is that it doesn't belong in the analytics branch, if you want to change it thats all well and good, just do it in a relevant branch. So Ideally please reset it back to how it was, create a new branch for modifying the new user workflow, and change it there.